packages/react-instantsearch/src/connectors/connectInfiniteHits.test.js

```diff
@@ -41,7 +42,7 @@ describe('connectInfiniteHits', () => {
       const hits2 = [{}, {}, {}, {}, {}, {}];
       const hits3 = [{}, {}, {}, {}, {}, {}, {}, {}];
       const res1 = getProvidedProps(null, null, {
-        results: { hits, page: 0, hitsPerPage: 6, nbPages: 10 },
+        results: { hits, page: 0, hitsPerPage: 6, nbPages: 10, index: 'index' },
```

Comment:
Should we use "indexName" like other parts of the lib? I would say any moment we speak about an index name we should use the "indexName" token as we do in every JS lib.